hack/verify-vendor.sh

```diff
@@ -3,6 +3,7 @@
 if [ "$IS_CONTAINER" != "" ]; then
   go mod vendor
   go mod verify
+  go mod tidy
```

Comment:
isn't the verify should be the last one?

Reply:
Right; thank you. I have eventually removed the call to `go mod tidy`, focusing on a more pressing need for `set -e` which I aim at merging with minimum controversy (which `tidy` could generate).

I will certainly consider your comment when I try to introduce `tidy` again.